dataflow/src/main/java/org/checkerframework/dataflow/cfg/AbstractCFGVisualizer.java

```diff
@@ -221,11 +214,42 @@ protected String loopOverBlockContents(
             Block bb, @Nullable Analysis<V, S, T> analysis, String separator) {
 
         List<Node> contents = addBlockContent(bb);
-        StringJoiner sjBlockContents = new StringJoiner(separator);
+        StringBuilder sbBlockContents = new StringBuilder();
         for (Node t : contents) {
-            sjBlockContents.add(visualizeBlockNode(t, analysis));
+            sbBlockContents.append(visualizeBlockNode(t, analysis)).append(separator);
+        }
+        return sbBlockContents.toString();
+    }
+
+    /**
+     * Helper method to visualize a node based on the analysis.
+     *
+     * @param t a node
+     * @param analysis the current analysis
+     * @param needEscape need to escape double quotes
+     * @return the visualization of the given node
+     */
+    protected String visualizeBlockNodeHelper(
```

Comment:
This change has been moved to typetools: https://github.com/typetools/checker-framework/pull/3395